### ISSUE TITLE
Custom colors reset each time Intellij restarts

### ISSUE THREAD

**Issue by erikdewit87** [one-dark/jetbrains-one-dark-theme#170]
**Describe the bug**

Each time Intellij starts custom colors are reset to their defaults. For example: I use custom console colors for Android logcat as their default colors are the same for debug/verbose/info and error/assert. Each time Intellij starts a new "One Dark Generated" color scheme is generated and applied which contains default colors.
![image](https://user-images.githubusercontent.com/1140942/86358811-6aa86000-bc70-11ea-9711-67f94767b21b.png)

The android logcat colors scheme is my example but I expect this behaviour for all (non-default?) plugins listed in "Editor" -> "Color Scheme"

**To Reproduce**

1. Set custom colors for Android Logcat under Preferences -> Editor -> Color scheme
2. Quit intellij 
3. Start intellij and open project
4. If editor/consoles switch from light to dark mode a "One Dark Generated" color scheme is applied
4. Android Logcat errors are back to default

**Expected Behavior**

I expect my custom colors to be restored after regenerating the "One Dark Generated" color scheme. See screenshot below

![image](https://user-images.githubusercontent.com/1140942/86360990-78131980-bc73-11ea-851b-9ec0efc62924.png)

**Environment**

I've attached the generated color scheme with custom colors for:
- `LOGCAT_DEBUG_OUTPUT`
- `LOGCAT_ASSERT_OUTPUT`
- `LOGCAT_VERBOSE_OUTPUT`
- `LOGCAT_INFO_OUTPUT`
- `LOGCAT_WARNING_OUTPUT`

[One Dark Generated.icls.zip](https://github.com/one-dark/jetbrains-one-dark-theme/files/4864298/One.Dark.Generated.icls.zip)

**Comment by Unthrottled** [comment]
Hey! That should have worked. I put code in that should detect if you import/duplicate the theme. I'll have to look into why it is not working. Thanks!

I could also add your logcat colors as well, if you want.

**Comment by erikdewit87** [comment]
> I could also add your logcat colors as well, if you want.

I think these are reasonable defaults for android logcat. 
Actually I copied them from: "Console Colors"->"Log console" ->"debug/verbose/info/warning/assert" 
The defaults point to "Console Colors"->"Console"->"Standard output" (white)

**Comment by Unthrottled** [comment]
@erikdewit87 

I have some work arounds for the time being (If you feel like some extra steps to get this to work).

If you could import the your icls file you attached, that will (should) trigger the `do not overwrite` code. (It worked for me)

<img width="448" alt="Screen Shot 2020-07-02 at 9 08 53 AM" src="https://user-images.githubusercontent.com/15972415/86369440-0f607880-bc44-11ea-9627-46f179c9b834.png">

If that does not work then you can edit the `one_dark_config.xml` (in the configuration directory) to include `<option name="customSchemeSet" value="true" />`

ex: 

```
<application>
  <component name="OneDarkConfig">
    <option name="attributesStyle" value="Bold" />
    <option name="customSchemeSet" value="true" />
    <option name="userId" value="2607a63d-0f5f-4b90-a369-4e9af25cf66f" />
    <option name="version" value="4.0.1" />
  </component>
</application>
```

[This is where you can find where the configuration directories are at.](https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs).

 That will stay in effect, unless you modify the config in the `One Dark Theme` settings menu, then it will reset `customSchemeSet`.

**Comment by Unthrottled** [comment]
> I think these are reasonable defaults for android logcat.

I will take that as a "yes"? 😄

**Comment by Unthrottled** [comment]
I decided to revert the configurable theme options in favor of a consistent and reliable theme user experience. Please upgrade to the stable v5.0.0.

I found other issues that are unacceptable to have at the moment. I will re-visit the configurable theme options in a later date, when I have more time to flesh out all of the quirks of making the platform have configurable themes. 

I have also included the Android Logcat colors in v5.0.0. Thanks for the suggestion!
 
Thank you for your patience and understanding!
